src/containers/ConnectorManagement.jsx

```diff
@@ -84,6 +85,7 @@ export default class ConnectorManagement extends Component {
               existingAccount={accounts.length ? accounts[selectedAccount] : null}
               alertSuccess={(messages) => this.alertSuccess(messages)}
               onCancel={() => this.gotoParent()}
+              isUnloading={this.state.isClosing}
```

Comment:
As we are already retrieving data from state l.73, we may also get `isClosing` from here.